config.sample.json

```diff
@@ -1,5 +1,6 @@
 {
     "api_key": "asasdkldasdlkaasdkfdsjadf",
+    "api_version": "2024-02-15",
```

Comment:
api_version should not be the configurable property unless your code is flexible enough to switch between different api versions.

Reply:
Sure, I'll make that change.